docker-compose.dev.yml

```diff
@@ -0,0 +1,42 @@
+# Docker-compose setup for local development
+#
+# Configurable Values:
+#   SEED_DB_VOLUME - name of docker volume to mount for Postgres
+#   SEED_MEDIA_VOLUME - name of docker volume to mount for the media directory
+#
+# Usage:
+#   docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
+version: '3.4'
+services:
+  db-postgres:
+    volumes:
+      - ref_seed_pgdata:/var/lib/postgresql/data
+  web:
+    image: seedplatform/seed:dev
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    command: python3 manage.py runserver --settings=config.settings.docker_dev 0.0.0.0:80
+    environment:
+      - DJANGO_SETTINGS_MODULE=config.settings.docker_dev
+    volumes:
+      - ref_seed_media:/seed/media
+      - ./:/seed/
+  web-celery:
+    image: seedplatform/seed:dev
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    command: watchmedo auto-restart --directory=/seed/seed/ --pattern=*.py --recursive -- /seed/docker/start_celery_docker.sh
+    environment:
+      - DJANGO_SETTINGS_MODULE=config.settings.docker_dev
+    volumes:
+      - ref_seed_media:/seed/media
+      - ./seed:/seed/seed
+volumes:
+  ref_seed_pgdata:
```

Comment:
Any reason this is called ref? I agree that it shouldn't be seed_pgdata as that would be what is on production. Perhaps `dev_seed_pgdata`?

Reply:
This is called ref because we are templating in the actual volume name, so this is just a "reference" to a volume (not the volume name itself), see here: 
https://github.com/SEED-platform/seed/pull/2423/files/b0fdbc1338b7af66d5f6075a919e8560185fc80a#diff-9542f82d64bbeebd91f6236324bfe199e9657e2cb1fd9779d5d6dcdcf9cd4de1R39